tests/system/action/topic/test_json_upload.py

```diff
@@ -0,0 +1,173 @@
+from openslides_backend.action.actions.topic.json_upload import ImportStatus
+from openslides_backend.permissions.permissions import Permissions
+from tests.system.action.base import BaseActionTestCase
+
+
+class TopicJsonUpload(BaseActionTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.set_models(
+            {
+                "meeting/22": {"name": "test", "is_active_in_organization_id": 1},
+            }
+        )
+
+    def test_json_upload_agenda_data(self) -> None:
+        response = self.request(
+            "topic.json_upload",
+            {
+                "meeting_id": 22,
+                "data": [
+                    {
+                        "title": "test",
+                        "agenda_comment": "testtesttest",
+                        "agenda_type": "hidden",
+                        "agenda_duration": 50,
+                    }
+                ],
+            },
+        )
+        self.assert_status_code(response, 200)
+        assert response.json["results"][0][0]["rows"][0] == {
+            "status": ImportStatus.NEW,
+            "error": [],
+            "data": {
+                "title": "test",
+                "meeting_id": 22,
+                "agenda_comment": "testtesttest",
+                "agenda_type": "hidden",
+                "agenda_duration": 50,
+            },
+        }
+
+    def test_json_upload_wrong_data(self) -> None:
+        response = self.request(
+            "topic.json_upload",
+            {"meeting_id": 22, "data": [{"title": "test", "wrong": 15}]},
+        )
+        self.assert_status_code(response, 400)
+        assert (
+            "data.data[0] must not contain {'wrong'} properties"
+            in response.json["message"]
+        )
+
+    def test_json_upload_empty_data(self) -> None:
+        response = self.request(
+            "topic.json_upload",
+            {"meeting_id": 22, "data": []},
+        )
+        self.assert_status_code(response, 400)
+        assert "data.data must contain at least 1 items" in response.json["message"]
+
+    def test_json_upload_results(self) -> None:
+        response = self.request(
+            "topic.json_upload",
+            {"meeting_id": 22, "data": [{"title": "test"}]},
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_exists(
+            "action_worker/1",
+            {
+                "result": {
```

Comment:
Missing the data for headers and statistics

Reply:
Why that? headers and statistics are not needed in the action worker.

Reply:
I don't think, that we should include headers and statistics in the action_worker, they are just wasted load. The topic import action doesn't need them. It just needs the `rows`.